data-prepper-plugins/opensearch/README.md

```diff
@@ -185,6 +185,9 @@ Besides common metrics in [AbstractSink](https://github.com/opensearch-project/d
 - `documentsSuccess`: measures number of documents successfully sent to ES by bulk requests including retries.
 - `documentsSuccessFirstAttempt`: measures number of documents successfully sent to ES by bulk requests on first attempt.
 - `documentErrors`: measures number of documents failed to be sent by bulk requests.
+- `documentNumberOfRetries`: measures number of times bulk requests are retried.
+- `documentNumberOfFailures`: measures number of times bulk requests are failed after retries.
```

Comment:
We already track document failures with the `documentErrors` metric.

From the code, it appears that you are tracking the number of bulk requests with any error.

What is the value of this metric?

If we do keep it, perhaps we should rename it to `bulkRequestErrors` to stay consistent.

Reply:
I agree that it looks like this is redundant. I will remove this.